action.yml

```diff
@@ -2,12 +2,12 @@ name: 'Running AQA test'
 description: 'Running AQA test'
 author: 'Sophia Guo'
 inputs:
-  version:              # jdk version
-    description: 'testing jdk version'
-    default: '8'
   jdksource:               # customized: if installed jdk by TheMrMilchmann/install-jdk or default: github hosted environment  
     description: 'git-hub hosted default or customized'
     required: false
+  version:              # jdk version
+    description: 'testing jdk version (Required when jdksource is default git-hub hosted.)'
```

Comment:
Do not need a - in github (link 6 and 9)
You will want to update line 8 as well to point to AdoptOpenJDK/install-jdk

otherwise LGTM